src/metrics.ts

```diff
@@ -271,6 +271,10 @@ export const MetricsDefaults: Record<string, any> = {
     textYOffset: 2,
   },
 
+  TupletNote: {
+    fontSize: 16,
+  },
+
```

Comment:
Please move this into Tuplet like

```js
Tuplet: {
    yOffset: 0,
    textYOffset: 2,
    note: {
      fontScale: 1/2,
    },
  },
```
I prefer to use `fontScale` rather than `fontSize` just in case someone changes the main `fontSize`.